internal/app/machined/pkg/system/services/bootkube.go

```diff
@@ -146,7 +147,11 @@ func (b *Bootkube) PreFunc(ctx context.Context, config runtime.Configurator) (er
 }
 
 // PostFunc implements the Service interface.
-func (b *Bootkube) PostFunc(config runtime.Configurator) (err error) {
+func (b *Bootkube) PostFunc(config runtime.Configurator, state events.ServiceState) (err error) {
```

Comment:
An example of us making use of this. Needed it for skipping the etcd update anyways.